content/en/docs/ops/best-practices/image-signing-validation/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Image Signing And Validation
+description: Describes how to use image signatures to verify the provenance of Istio images.
+weight: 35
+aliases:
+    - /docs/ops/best-practices/image-signing-validation/
+keywords: [install]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This page describes how to use [Cosign](https://github.com/sigstore/cosign) to
+validate the provenance of Istio image artifacts.
+
+Cosign is a tool developed as part of the
+[sigstore](https://www.sigstore.dev/https://www.sigstore.dev/) project which
+simplifies signing and validation of signed Open Container Initiative (OCI) artifacts,
+such as docker images.
+
+In Istio, we sign all officially published docker images as part of our release
+process using our own private key. End users can then verify these images using
+the process described below.
+
+This process is suitable for either manual execution or integration with build
+or deployment pipelines for automated verification of artifacts.
+
+## Prerequisites
+
+Before you begin, please do the following:
+
+1. Download the latest
+   [Cosign](https://github.com/sigstore/cosign/releases/latest) build for your
+   architecture, as well as its signature.
+1. Validate the cosign binary signature: 
+   {{< text bash >}}
+$ openssl dgst -sha256 \
+	-verify <(curl -ssL https://raw.githubusercontent.com/sigstore/cosign/main/release/release-cosign.pup) \
+	-signature /path/to/cosign.sig \
+	/path/to/cosign-binary
+    {{< /text >}}
+1. Run `chmod +x` to make the Cosign binary executable
+
+## Validating Image
+
+To validate a docker image, do the following:
+
+{{< text bash >}}
+$ ./cosign-binary verify --key {{< static "/misc/istio-key.pub" >}} {{< istio_docker_image "pilot" >}}
```

Comment:
The pub key isn't working correctly, at least if I look at the deploy/netlify test details link, https://deploy-preview-10775--preliminary-istio.netlify.app/latest/docs/ops/best-practices/image-signing-validation/

Reply:
It looks like it is setting `$.Site.Page.BaseURL` to `/latest/` rather than the full URL.